doc/install/install-openshift.md

```diff
@@ -287,9 +261,36 @@ kubectl get installplan -n kuadrant-system -o=jsonpath='{.items[0].status.phase}
 
 After some time, this command should return `complete`.
 
-### Step 9 - Configure Kuadrant
 
-To configure your Kuadrant deployment, enter the following command:
+#### Redis credentials for storage of rate limiting counters
+
+In this installation we will show how to configure ratelimiting counters to be stored in redis. Before we go further we need to setup a redis secret to use later:
+
+```bash
+kubectl -n kuadrant-system create secret generic redis-config \
+  --from-literal=URL=$REDIS_URL
+```
+
+#### Set up a DNSProvider
+
+The example here is for AWS Route 53. It is important the secret for the DNSProvider is setup in the same namespace as the gateway.
+
+```bash
+kubectl create ns ingress-gateway
+```
+
+```bash
+kubectl -n ingress-gateway create secret generic aws-credentials \
+  --type=kuadrant.io/aws \
+  --from-literal=AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID \
+  --from-literal=AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+```
+
+For more details on other providers take a look at [DNS Providers](https://github.com/Kuadrant/dns-operator/blob/main/docs/provider.md)
```

Comment:
```suggestion
For more details on other providers take a look at [DNS Providers](dns-operator/blob/main/docs/provider.md)
```

should expand to the right link  (in docs, rather than booting them out to GH)

Reply:
hmm yeah but then if you come across this in GH the link wont work right?

Reply:
maybe we add that to docs.kuadrant.io first

Reply:
> maybe we add that to docs.kuadrant.io first

that makes sense yeah